docs/tutorials/basic/module.md

```diff
@@ -141,15 +164,27 @@ mod.fit(train_iter,
 
 ## Intermediate-level Interface
 
-We already saw how to use module for basic training and inference. Now we are
-going to see a more flexible usage of module. Instead of calling
-the high-level `fit` and `predict` APIs, one can write a training program with the intermediate-level APIs
-`forward` and `backward`.
+We can also use module's intermediate-level APIs to do training and inference.
```

Comment:
I feel we should first explain what are the steps involved(the below) and then introduce the high-level abstraction(fit) explicitly calling out the defaults since fit has a lot of arguments.
we should also link to the fit API documentation http://mxnet.io/api/python/module.html#mxnet.module.BaseModule.fit